ff/src/fields/models/fp6_3over2.rs

```diff
@@ -24,6 +24,8 @@ impl<P: Fp6Config> CubicExtConfig for Fp6ConfigWrapper<P> {
     type BaseField = Fp2<P::Fp2Config>;
     type FrobCoeff = Fp2<P::Fp2Config>;
 
+    const PRECOMP: Option<SqrtPrecomputation<CubicExtField<Self>>> = None;
```

Comment:
Let's change this to `const PRECOMP: ... = P::PRECOMP;`, and add a `const PRECOMP: Option<SqrtPrecomputation<CubicExtField<Self>>> = None;` to `Fp6Config`. This way, if someone downstream wishes to specify a  sqrt algorithm, they can do it easily.

This would also allow us to write `Fp6<Self>` instead of `CubicExtField<Self>`

Reply:
Also, let's call this `SQRT_PRECOMP` like elsewhere.